src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -133,7 +133,12 @@ public void handle(Endpoint<?> endpoint) {
                     }
 
                     this.commit(offsetData, status -> {
-                        sendConsumerCommitOffsetResponse(httpServerRequest.response());
+                        if (status.succeeded()){
+                            sendConsumerCommitOffsetResponse(httpServerRequest.response(), true);
+                        } else {
+                            sendConsumerCommitOffsetResponse(httpServerRequest.response(), false);
+                        }
```

Comment:
why not just passing `status.succeeded()` instead of the `if` and then passing `true` or `false` ? :-)